services/secret-service/doc/openapi.yaml

```diff
@@ -299,22 +339,87 @@ components:
         - clientSecret
         - refreshTokenUrl
       properties:
-        name:
-          type: string
         clientId:
           type: string
         clientSecret:
           type: string
-        refreshTokenUrl:
+        redirectUri:
           type: string
+        endpoint:
+          type: object
+          properties:
+            auth:
+              type: string
+            token:
+              type: string
+            userinfo:
+              type: string
+            revocation:
+              type: string
+            endSession:
+              type: string
+          required:
+            - auth
+            - token
+        predefinedScope:
+          type: string
+        mappings:
+          type: object
+          properties:
+            scope:
+              type: object
+              properties:
+                key:
+                  type: string
+              required:
+                - key
+            externalId:
+              type: object
+              properties:
+                source:
+                  type: string
+                  enum: [id_token, access_token, userinfo]
+                key:
+                  type: string
+              required:
+                - source
+                - key
+      example: {
+        "_id": "5c06ee51120e789235d9dc19",
+        "endpoint": {
+          "auth": "https://accounts.google.com/o/oauth2/v2/auth?prompt=consent&scope={{scope}}&access_type=offline&include_granted_scopes=true&state={{state}}&redirect_uri={{redirectUri}}&response_type=code&client_id={{clientId}}",
+          "token": "https://www.googleapis.com/oauth2/v4/token",
+          "userinfo": "https://www.googleapis.com/oauth2/v4/userinfo"
+        },
+        "clientId": "myclientid",
+        "clientSecret": "shhhhhhhh",
+        "redirectUri": "http://localhost:9001/api/v1/callback",
+        "mappings": {
+          "externalId": {
+            "source": "id_token",
+            "key": "sub"
+          },
+          "scope": {
+            "key": "scope"
+          }
+        },
+        "type": "OA2_AUTHORIZATION_CODE",
+        "name": "example google oAuth2",
+        "owner": [
+        {
+          "entityId": "u4",
+          "entityType": "USER"
+        }
+        ],
+      }
     MutableOAuth1Client:
       required:
         - consumerKey
         - consumerSecret
         - nonce
         - signature
         - signatureMethod
-        - verifier
+        - version
```

Comment:
`verifier` stands for the `oauth_verifier` param defined in the OAuth1 spec. It was replaced by `version`. Why is and what stands `version` for?

Reply:
You have `verifier` as required, but `version` on line 331 in your version v.0.0.5. Probably a typo.
We'll change both to verifier.